presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -511,6 +514,12 @@ protected void setup(Binder binder)
         driftServerBinder(binder).bindService(ThriftTaskService.class);
         driftServerBinder(binder).bindService(ThriftServerInfoService.class);
 
+        // Async page transport
+        newMapBinder(binder, String.class, Servlet.class, TheServlet.class)
+                .addBinding("/v1/task/async/*")
```

Comment:
wow :). Where is the binding for non-async end-point?

Reply:
That is handled by `@Path` annotation in JAX-RS